### ISSUE TITLE
Player title bar at top

### ISSUE THREAD

**Issue by tim-sebring** [eniklas/gamatrix-gog#15]
Would be cool for the grid to have the player headers at the top remain up top when scrolling, so you don't have to scroll back up to remember who is in what column.

**Comment by eniklas** [comment]
Beauty idea, I've found that annoying too when the title bar scrolls off the top. I've thought about repeating the bar every so many rows, or just putting the usernames in the cells every so often (I've tried putting them in every row but it's distracting). Looks like a sticky bar [would require JavaScript](https://www.w3schools.com/howto/howto_js_sticky_header.asp), which I'm not necessarily opposed to.
